src/main/java/duke/ExitCommand.java

```diff
@@ -0,0 +1,16 @@
+package duke;
```

Comment:
Might want to consider shifting some of these classes into separate packages

Reply:
fixed!